File: @LV1/1.4/heartrate/heartrate.cpp
#include <cstdio>

int main() {
    int n;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        double b, p;
        scanf("%lf %lf", &b, &p);

        double bpm = 60 * b / p;
        double abpm = 60 / p;
        printf("%.6lf %.6lf %.6lf\n", bpm - abpm, bpm, bpm + abpm);
    }
}

File: @LV1/1.4/heartrate/heartrate.hs
import           Control.Monad

main :: IO ()
main = do
  n <- readLn
  replicateM_ n $ do
    [b, p] <- map read . words <$> getLine
    let bpm  = 60 * b / p
        abpm = 60 / p
    putStrLn $ show (bpm - abpm) ++ show bpm ++ show (bpm + abpm)
